### DISCUSSION TITLE
CMS state mandate file generation -  which folder do we need to refer(Example/Schema)?

### DISCUSSION THREAD

**Discussion by vipulll** [CMSgov/price-transparency-guide#477] [Q&A] [answered]
Hi All,

1. Do we need to generate the CMS file as per "CMSgov/price-transparency-guild/Example/in-network-rates" folder from git ?

2. For In-network file generation.
File name:- in-network-rates-fee-for-service-single-plan-sample.json 
Below is a link to that page.
Link:- https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json

What do we need to add to the below JSON list? 
{
    "provider_group_id": 2,
    "location": "https://www.cms.gov/some_provider_group.json"
  }
![Question](https://user-images.githubusercontent.com/29914377/168049189-e3eae304-5535-42e5-ab2b-e9b5e58ae225.PNG)

3. from where can I know the details about which fields are required and which are not required?

**Comment by shaselton-usds** [comment, marked as answer]
@vipulll Requirements can be found in the [schemas directory](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas). Implementation examples of those requirements can be found the [example directory](https://github.com/CMSgov/price-transparency-guide/tree/master/examples).

I'm not quire sure what you're asking for question number 2. A lot of information surrounding the external provider reference can be found in [January 11th webinar](https://www.cms.gov/healthplan-price-transparency/resources/webinars)
